### PR TITLE
Break up operations manual into topics files

### PR DESCRIPTION
**Pull request by ajw1970**

### CODE REVIEW

**Comment by ajw1970** [comment]
@briandominick , this is a smaller example that might be easier to discuss. 

all of these sub-topics would apply to either an engel or ipi branded coil line with the exception of hand-feeding-blanks.adoc which is currently brand specific to ipi.

**Comment by ajw1970** [comment]
Would this IPI027V02_PDoperation.adoc with the includes go away and be replaced with a _manifest file somewhere?

Would I then also need that introduction paragraph to be placed in an adoc file or would that type of thing go somewhere else?

File: ProFabriduct/Operation/e-stop-recovery.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+== Recovering from Emergency Shutdown
```
Comment (briandominick) [inline ProFabriduct/Operation/e-stop-recovery.adoc:1]: Best practice is to always start files with level 0 headers (1 `=` sign).

Then we adjust the level where we `include` the file. Consider that for web pages, we want level 0, and for chapters, we want level 1 (`==`), and sometimes a topic will get used in more than one place and might have a different contextual level. AsciiDoc offers `:leveloffset:` which can be `+1`, `+2`, or `-1`,  etc. So we set the starting level contextually at the point of publishing.

**Review by briandominick — CHANGES_REQUESTED** [review]
Other than one note about header levels, this looks good.

To answer your questions:

> Would this IPI027V02_PDoperation.adoc with the includes go away and be replaced with a _manifest file somewhere?

Yes, we will generate these index files on the fly from a template drawing from each product's `_manifest.yml` file. This is a good model for now, though, so let's keep it.

> Would I then also need that introduction paragraph to be placed in an adoc file or would that type of thing go somewhere else?

We can include anything we want in the template that generates the `.adoc` files. But we'll also be able to use conditionals and variables, so we can source this stuff anywhere and transclude it on a per-product or per-volume basis.

**Pull request merged**